package.json

```diff
@@ -42,10 +42,15 @@
       "import": "./dist/argentMobile.js",
       "require": "./dist/argentMobile.cjs"
     },
-    "./argentCompound": {
-      "types": "./dist/argentCompound.d.ts",
-      "import": "./dist/argentCompound.js",
-      "require": "./dist/argentCompound.cjs"
+    "./argent": {
+      "types": "./dist/argent.d.ts",
+      "import": "./dist/argent.js",
+      "require": "./dist/argent.cjs"
+    },
+    "./argentX": {
+      "types": "./dist/argentX.d.ts",
```

Comment:
lost some details, are we exporting ax/braavos just in case a dapp just needs those 2 right?